.github/workflows/reusable-docker-build-qa.yml

```
@@ -0,0 +1,637 @@
+        description: 'Ref for liquibase/build-logic checkout; override during contract-churn branches'
+        required: false
+        type: string
+        default: 'main'
+    outputs:
+      image_tag:
+        description: 'Tag used for the pushed image (branch name or RC version)'
+        value: ${{ jobs.build-qa.outputs.image_tag }}
+      ghcr_ref:
+        description: 'Full GHCR image reference including tag'
+        value: ${{ jobs.build-qa.outputs.ghcr_ref }}
+    secrets: inherit
```

actionlint (1.7.12)

[error] "secrets" section is scalar node but mapping node is expected

(syntax-check)